.github/workflows/update.yml

```diff
@@ -33,8 +35,45 @@ jobs:
             git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
             git config --global user.name "github-actions[bot]"
   
-      - name: Run nix-update on each package
-        run: nix-shell -p nix-update --run 'sh update-packages.sh'
-        
-      - name: Run git log to show changes
-        run: git log -10
+      - name: Run nix-update on each package, push to a branch, and open a PR if it doesn't yet exists
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+            declare -a pkgs=(
+              "addrman-observer"
+              "fork-observer"
+              "miningpool-observer"
+              "peer-observer"
+              "github-metadata-mirror"
+              "github-metadata-backup"
+              "transactionfee-info-backend"
```

Comment:
How to keep this in sync with the actual packages?

Reply:
Done with `pkgs=($(nix flake show --json | jq -r '.packages."x86_64-linux" | keys[] ' | grep -v bitcoind-tracing))`